File: 88-merge-sorted-array/88-merge-sorted-array.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        vector<int>d(nums1.begin(),nums1.begin()+m);
        for(int i=0;i<n;i++)
            d.push_back(nums2[i]);
        sort(d.begin(),d.end());
       for(int i=0;i<d.size();i++)
           nums1[i]=d[i];
    }
};

File: 88-merge-sorted-array/88-merge-sorted-array.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
       Arrays.sort(nums1,0,m);
       Arrays.sort(nums2);
        int j=0;
        for(int i=m;i<nums1.length;i++)
        {
            nums1[i]=nums2[j];
            j++;
        }
        Arrays.sort(nums1);
        return;
    }
}